packages/experimental/Avatar/ios/MSFAvatarViewManager.m

```diff
@@ -0,0 +1,76 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+
+@import FluentUI;
+
+@implementation RCTConvert (MSFAvatarViewAdditions)
+
+RCT_ENUM_CONVERTER(MSFAvatarSize, (@{
+	@"xSmall": @(MSFAvatarSizeExtraSmall),
+	@"small": @(MSFAvatarSizeSmall),
+	@"medium": @(MSFAvatarSizeMedium),
+	@"large": @(MSFAvatarSizeLarge),
+	@"xLarge": @(MSFAvatarSizeExtraLarge),
+	@"xxLarge": @(MSFAvatarSizeExtraExtraLarge),
+}), MSFAvatarSizeSmall, integerValue);
+
+RCT_ENUM_CONVERTER(MSFAvatarStyle, (@{
+	@"circle": @(MSFAvatarStyleCircle),
+	@"square": @(MSFAvatarStyleSquare),
+}), MSFAvatarStyleCircle, integerValue);
+
+RCT_ENUM_CONVERTER(MSFAvatarFallbackImageStyle, (@{
+	@"onAccentFilled": @(MSFAvatarFallbackImageStyleOnAccentFilled),
+	@"outlined": @(MSFAvatarFallbackImageStyleOutlined),
+	@"primaryFilled": @(MSFAvatarFallbackImageStylePrimaryFilled),
+	@"primaryOutlined": @(MSFAvatarFallbackImageStylePrimaryOutlined),
+}), MSFAvatarFallbackImageStyleOutlined, integerValue);
+
+RCT_ENUM_CONVERTER(MSFPresence, (@{
+	@"none": @(MSFPresenceNone),
+	@"available": @(MSFPresenceAvailable),
+	@"away": @(MSFPresenceAway),
+	@"busy": @(MSFPresenceBusy),
+	@"doNotDisturb": @(MSFPresenceDoNotDisturb),
+	@"outOfOffice": @(MSFPresenceOutOfOffice),
+	@"offline": @(MSFPresenceOffline),
+	@"unknown": @(MSFPresenceUnknown),
+	@"blocked": @(MSFPresenceBlocked),
+}), MSFPresenceNone, integerValue);
+
++ (MSFAvatarData *)MSFAvatarData:(id)json
+{
+	return [[MSFAvatarData alloc]initWithPrimaryText:[RCTConvert NSString:json[@"primaryText"]]
+                                     secondaryText:[RCTConvert NSString:json[@"secondaryText"]]
```

Comment:
nit- misaligned with first line's colon

Reply:
First line uses a tab, rest of the lines uses spaces assuming tab width is 2 spaces. I think github assumes a different tabwidth. I looked into what we could do to address that, and maybe we could add a .editorconfig https://stackoverflow.com/questions/8833953/how-to-change-tab-size-on-github/23522945#23522945

Reply:
#523 for this one